src/controllers/controller.radar.js

```diff
@@ -139,8 +148,9 @@ module.exports = DatasetController.extend({
 
 			// Desired view properties
 			point._model = {
-				x: x, // value not used in dataset scale, but we want a consistent API between scales
-				y: y,
+				x,
+				y,
+				r: pointPosition.r,
```

Comment:
`r` is unclear as a variable name. Can we give it a longer name to make it clear what it represents?

Reply:
See https://github.com/chartjs/Chart.js/pull/6795#issuecomment-562971024

Reply:
Any suggestions for good variable name for the radial axis are welcomed.

Reply:
How about `axis`?

Reply:
Is that variable used? The only place I saw `interpolate` being called was in `interpolatedLineTo`, which doesn't use it. Maybe we can remove it?

Reply:
Nevermind, you still need it, but there's another variable I think can be removed that I just commented on. But I don't think you need to pass the actual axis. Maybe just a boolean `isRadial` would be enough?

Reply:
How about calling it `axis`?

Reply:
Not sure if it is any more obvious:
```js
scales: {
  myRadial: {
    axis: 'axis', 
   ... 
  }
}
```

Reply:
I would have called it `α`, but multibytes dont seem to work well.

Reply:
I thought this is just an internal model. I'm confused how the options example you shared is related

Reply:
I thought this is just an internal model. I'm confused how the options example you shared is related

Reply:
`Scale`'s `axis` property is used to determine what `point`'s property should be used to match the two lines together (that are filled between). For radial it needs to be the angle.

Some new property could be used to determine that too, but it it would mean another property with value `'x'` / `'y'` for the "normal" scales.